### DISCUSSION TITLE
v5 Select选择器圆角问题

### DISCUSSION THREAD

**Discussion by WT-Road** [ant-design/ant-design#40062] [Q&A]
默认的6px圆角被覆盖,但是全局搜索.ant-select-selector并没有设置border-radius: 0px;
然后全局搜索注释掉所有的.ant-select-selector样式还是被覆盖,很困惑的并没有设置圆角,但确是0px
![image](https://user-images.githubusercontent.com/14371925/210934100-51b75dbc-e405-4bc5-b48b-50b4ef9dcf87.png)

**Comment by Wxh16144** [comment]
有什么进展么？

**Reply by WT-Road** [reply]
> 有什么进展？

Input有圆角,但是Select却是0,也找不到是原因导致的覆盖

**Reply by WT-Road** [reply]
已解决,移除了"@ant-design/pro-layout": "^6.38.22",样式正常了
